PebbleSharp.Core/BlobDB/BlobDBClient.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace PebbleSharp.Core.BlobDB
+{
+	public class BlobDBClient
+	{
+		private Pebble _pebble;
+		private Random _random;
```

Comment:
Though unlikely to cause any issues. It might be base to declare this as 
```C#
private static readonly Random _random = new Random();
```
And remove the initialization from the constructor. In the unlikely case where two BlobClient classes are created at the same time, their random number generators would get created with identical seeds.